config/application.rb

```diff
@@ -1,31 +1,32 @@
-require File.expand_path('../boot', __FILE__)
+require_relative "boot"
 
-require 'rails/all'
+require "rails/all"
 
-if defined?(Bundler)
-  Bundler.require *Rails.groups(:assets => %w(development test))
-end
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
 
 module IndyhackersRails
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 6.1
```

Comment:
I chose to forgo the "here are the defaults for the new version, enable them when you want" helper file and went straight to invoking all defaults for the latest version. Nothing really stood out as overly concerning for this app.